moshi-sealed/metadata-reflect/src/main/resources/META-INF/com.android.tools/proguard/moshi-sealed-metadata-reflect.pro

```diff
@@ -0,0 +1,16 @@
+# When editing this file, update the following files as well:
+# - META-INF/com.android.tools/r8-from-1.6.0/moshi-sealed-metadata-reflect.pro
+# - META-INF/com.android.tools/r8-upto-1.6.0/moshi-sealed-metadata-reflect.pro
+# - META-INF/proguard/moshi-sealed-metadata-reflect.pro
+# Keep Metadata annotations so they can be parsed at runtime.
+-keep class kotlin.Metadata { *; }
+
+# Keep default constructor marker name for lookup in signatures.
+-keepnames class kotlin.jvm.internal.DefaultConstructorMarker
+
+# Keep implementations of service loaded interfaces
+-keep interface com.squareup.moshi.kotlin.kotlinx.metadata.impl.extensions.MetadataExtensions
```

Comment:
I've copied this file from `moshi-metadata-reflect` but I'm not sure if the package name here is correct, doesn't this incorrectly refer to the kotlin-reflect packages from square/moshi?

Reply:
Heh, good catch. I'll fix both of these separately